cdap-docs/reference-manual/source/licenses/cdap-standalone-dependencies.csv

```diff
@@ -59,9 +57,11 @@ javax.servlet.javax.servlet-api-3.0.1.jar
 javax.transaction.jta-1.1.jar
 javax.ws.rs.javax.ws.rs-api-2.0.jar
 javax.xml.bind.jaxb-api-2.1.jar
+jdk.tools.jdk.tools-1.7.jar
 jline.jline-2.12.jar
 joda-time.joda-time-2.1.jar
 log4j.log4j-1.2.16.jar
+native
```

Comment:
"native" is one of the dependencies in Standalone dependencies. IIANM, we can ignore it. I will file an issue about changing the license code to ignore it if so. I will check with Sree. It doesn't show up in the previous dependencies; let me see if I can fix it.